src/sentry/api/endpoints/organization_details.py

```diff
@@ -226,6 +226,10 @@ def put(self, request, organization):
                 data=organization.get_audit_log_data(),
             )
 
+            result = serializer.object
```

Comment:
Do we want to trigger this from the endpoint itself?

My concern with this is it's potentially noisy? Like, if a user toggled it on/off/on, we'd be blasting out emails to everyone multiple times.

It's probably fine for now though until someone complains or abuses it.

We should mention in the UI what will happen when this is enabled though. Something like what GitHub says:

> Members, billing managers, and outside collaborators who do not have two-factor authentication enabled for their personal account will be removed from the organization and will receive an email notifying them about the change.

Just so it's clear that we'll be emailing users when this happens and they will be locked out.

Reply:
Side note, it feels like this should go under `OrganizationSerializer.save()` instead.

Reply:
@mattrobenolt I was wondering about that. I noticed a few places in the code where something similar was done. That said, I can put it anywhere. Is the ideal place the serializer save? I hadn't seen any places do it there.